pkg/cmd/kafka/create/create.go

```diff
@@ -225,7 +234,7 @@ func runCreate(opts *options) error {
 			return f.Localizer.MustLocalizeError("kafka.create.error.noInteractiveMode")
 		}
 
-		payload, err = promptKafkaPayload(opts, *userInstanceType)
+		payload, err = promptKafkaPayload(opts, userInstanceType)
```

Comment:
The interactive mode  needs more changes, currently the user will need to specify `BillingModel` as flag before starting the interactive mode.